merino/jobs/wikipedia_indexer/indexer.py

```diff
@@ -88,6 +114,14 @@ def index_from_export(self, total_docs: int, elasticsearch_alias: str):
         else:
             raise Exception("Could not create the index")
 
+    def _should_filter(self, categories: set[str]) -> bool:
+        """Return True if we do not want to index this document."""
+        # Takes the intersection of the categories and blocklist.
+        # If there exists some shared categories with the blocklist,
+        # then skip processing this document.
+        overlapping_categories = categories & self.blocklist
+        return overlapping_categories != set()
```

Comment:
suggestion (non-blocking):

```suggestion
        return not self.blocklist.isdisjoint(categories)
```

...Or maybe renaming this to `_should_index`, and inverting the conditions?

Reply:
Good idea!